src/app/client_sdk/client_sdk.ml

```diff
@@ -84,14 +84,14 @@ let _ =
              Mina_base_nonconsensus.Signed_command_payload.dummy
            in
            let signature =
-             Mina_base_nonconsensus.Signed_command.sign_payload sk
-               dummy_payload
+             Mina_base_nonconsensus.Signed_command.sign_payload ~mainnet:true
```

Comment:
would you mind defining a constant bool somwhere in client_sdk/ and referring to that instead of passing the `true` literal everywhere. This way, at least we can quickly edit the implementation whenever we want to bake dev-compatible client-sdks (I think that's sufficient enough and worth landing this)